unstructured_inference/inference/layout.py

```diff
@@ -153,6 +153,23 @@ def from_image_file(
             pages.append(page)
         return cls.from_pages(pages)
 
+    def clean_pdfminer_inner_elements(self):
+        """Clean pdfminer elements from inside tables and stores them in pdfminer_inner_text
+        property of the elements"""
+        for page in self.pages:
+            tables = [e for e in page.elements if e.type == "Table"]
+            for i, element in enumerate(page.elements):
+                if element.source == Source.PDFMINER:
```

Comment:
lets do inverse condition then `continue` here to reduce indentation of the code; its a good practice to improve code readability in general. Especially here we have so many levels of indentation